src/edu-benefits/schema.js

```diff
@@ -280,7 +280,7 @@ module.exports = {
           }
         }
       },
-      required: ['commissionYear', 'rotcScholarshipAmounts']
+      required: ['rotcScholarshipAmounts']
```

Comment:
ROTC Scholarship shouldn't be required either. Am I reading that correctly?

Yes/no exposes it and indicates, but we (unintuitively [<-- not a word?]) don't want to use that to require any fields.